template/README.md

```diff
@@ -24,6 +24,8 @@ Any valid configuration added to these directories will be added to their respec
 #### Nomad ACLs
 Refer to [config variations](#config-variations) on how to toggle ACLs in nomad.
 
+When ACLs in Nomad are on the bootstrap management token will be available in vault under `secret/nomad/management-token` with the two key-value pairs `accessor-id` and `secret-id`. `secret-id` is the token itself. This can be accessed from inside the vagrant box with `vault kv get secret/nomad-bootstrap-token`, or by going to vault's UI on `localhost:8200` and signing in with the root token.
```

Comment:
Option with CLI, I would suggest run `vault` cli from localhost,
example:
```bash
vagrant ssh -c "VAULT_ADDR=http://127.0.0.1:8200 vault kv get secret/nomad-bootstrap-token"
```

But 
```
Error making API request.

URL: GET http://127.0.0.1:8200/v1/sys/internal/ui/mounts/secret/nomad-bootstrap-token
Code: 500. Errors:

* missing client token
```

Option vie UI works fine.

Reply:
Ah, I've probably set VAULT_TOKEN in my environment. I'll add that to the doc, thanks.

Reply:
Fixed

Reply:
Pay attention that I have also marked `VAULT_ADDR=http://127.0.0.1:8200`, because by default cli use `https` not `http`

Reply:
Crap, yep, you're right.

Reply:
There we go (I hope, haha)!